### PR TITLE
Switch to SVG

### PR DESCRIPTION
**Pull request by fejta**
* Switch to SVG for higher quality text, using GCS's gzip content stream feature to reduce file size to 300K instead of 4M.
* Copy image to old uri until next submit queue push
* Change `7 PRs queued` to `Queued PRs: 7` to better match labels on other graphs

Example output:
* http://storage.googleapis.com/kubernetes-test-history/k8s-queue-health-2.png, copied from
* http://storage.googleapis.com/kubernetes-test-history/k8s-queue-health-3.png

### CODE REVIEW

**Comment by ixdy** [comment]
Both of your linked examples are PNGs - was that intentional?

**Comment by fejta** [comment]
Yes, after this is in we're going to switch the extension of the uri to match the new content-type.

File: queue-health/graph.py [hunk 1]
```diff
@@ -220,14 +222,17 @@ def render_forever(history_uri, png_uri):
...
+            os.path.basename(img_uri), mode='wb', fileobj=buf) as compressed:
+            render(history.split('\n')[-60*24*21:], compressed)  # Last 21 days
 
-        print >>sys.stderr, 'Copy buffer to %s...' % png_uri
+        print >>sys.stderr, 'Copy buffer to %s...' % img_uri
         proc = subprocess.Popen(
             ['gsutil', '-q',
-             '-h', 'Content-Type:image/png',
+             '-h', 'Content-Type:image/svg+xml',
              '-h', 'Cache-Control:public, max-age=60',
```
Comment (lavalamp) [inline queue-health/graph.py]: that's 60 seconds, right?

Comment (fejta) [inline queue-health/graph.py]: Yes

**Comment by lavalamp** [comment]
lgtm but I'll let Ryan weigh in :)

**Pull request merged**